packages/luma/src/luma-image-model.ts

```diff
@@ -86,6 +76,10 @@ export class LumaImageModel implements ImageModelV2 {
       });
     }
 
+    // remove non-request options from providerOptions
+    const { pollIntervalMillis, maxPollAttempts, ...providerRequestOptions } =
+      providerOptions.luma ?? {};
```

Comment:
When providers want to support options that should not be passed through to the API calls then they have to manually extract them before sending the request. As an alternative we could use a special key like `providerOptions._.*` for options that should not be sent as part of the request body.